apps/web/lang/de.json

```diff
@@ -22,17 +22,17 @@
         },
         "consentCookie": {
           "name": "consent-cookie",
-          "provider": "plentysystems AG",
+          "provider": "plentyONE AG",
```

Comment:
All places with the AG stay the way they are. PlentyONE is the product, plentysystems AG is the legal name of the company